identity/aziot-identityd/src/lib.rs

```diff
@@ -555,14 +549,14 @@ mod tests {
 		}
 	}
 
-	#[test]
-	fn init_identities_with_empty_args_exits_early() {
+	#[tokio::test]
+	async fn init_identities_with_empty_args_exits_early() {
 		let server = Server::new(
 			make_empty_settings(),
 			Box::new(|_| Ok(AuthId::Unknown)),
 			Box::new(|_| Ok(true))
 		).unwrap();
 
-		assert!(wait!(server.init_identities(BTreeSet::new(), BTreeSet::new())).is_ok());
+		assert!(server.init_identities(BTreeSet::new(), BTreeSet::new()).await.is_ok());
```

Comment:
You can get a better panic message with `let _ = result.unwrap();` compared to `assert!(result.is_ok());`, because the panic message will include the `Err` value.

Reply:
Yeah that's better. I'll change it.
```
---- tests::init_identities_with_empty_args_exits_early stdout ----
thread 'tests::init_identities_with_empty_args_exits_early' panicked at 'called `Result::unwrap()` on an `Err` value: DeviceNotFound', identity/aziot-identityd/src/lib.rs:563:28
```

Reply:
Updated